jwst/extract_1d/extract.py

```diff
@@ -1180,18 +1229,18 @@ def offset_from_offset(self, input_model, slit, verbose):
             The pixel coordinate of the target in the cross-dispersion
             direction, at the middle of the spectrum in the dispersion
             direction.
-
-        verbose : bool
-            If True, log messages.
         """
 
+        targ_ra, targ_dec = self.get_target_coordinates(input_model, slit)
+        if targ_ra is None or targ_dec is None:
+            return (0., None)
```

Comment:
```suggestion
            return 0., None
```
Parentheses here aren't necessary